L/LibTracyClient/LibTracyClient@0.10.0/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "LibTracyClient"
+version = v"0.10.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/wolfpld/tracy.git",
+              "37aff70dfa50cf6307b3fee6074d627dc2929143"), # v0.10
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/tracy*/
+if [[ "${target}" != *-mingw* ]]; then
+    echo "target_compile_definitions(TracyClient PUBLIC __STDC_FORMAT_MACROS)" >> CMakeLists.txt
+else
+    echo "target_compile_definitions(TracyClient PUBLIC WINVER=0x0602 _WIN32_WINNT=0x0602)" >> CMakeLists.txt
```

Comment:
Why?  We already define these macros at _newer_ values: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/45acaa5c78a091ef75e5daa5de7a8676600509a5/src/Runner.jl#L555

Reply:
No idea, this was added by @topolarity for v0.9 in https://github.com/JuliaPackaging/Yggdrasil/pull/6087 and I just copy-pasted.
Do you mean that I can just remove this 5 lines now ?

Reply:
It's also in line 26 of [T/TracyProfiler/build_tarballs.jl](https://github.com/JuliaPackaging/Yggdrasil/pull/11089/files#diff-f7f4dadb3426699fc54f558c4a049dcb6dfb005780cf08cfbfb5e262da26482e)

Reply:
This was probably copied from https://github.com/JuliaLang/julia/blob/master/deps/libtracyclient.mk (which will also need an update together with this Yggdrasil recipe)